impeller/scene/importer/mesh.fbs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+namespace impeller.fb;
+
+struct Vec2 {
+  x: float;
+  y: float;
+}
+
+struct Vec3 {
+  x: float;
+  y: float;
+  z: float;
+}
+
+struct Color {
+  r: float;
+  g: float;
+  b: float;
+}
+
+struct Vertex {
+  position: Vec3;
+  normal: Vec3;
+  tangent: Vec3;
+  texture_coords: Vec2;
+}
+
+table Mesh {
```

Comment:
We'll probably have to sketch this out some more but this is a start!

Reply:
Indeed, one triangle mesh to fake it till we make it. Materials, fuller scene description, bones, weights, and animation will all be necessary soon in my estimation. :)